impeller/entity/shaders/gradients/fast_gradient.vert

```diff
@@ -18,6 +18,7 @@ in mediump vec4 color;
 out mediump vec4 v_color;
 
 void main() {
+  gl_PointSize = 1.0;
```

Comment:
Future opt - what does gl_PointSize render if the size > 1.0? Does it mimic any of the cap-styled points we need? Could we use this for non-hairline points of particular cap-styles?

There is also apparently a glLineWidth setting.

Reply:
gl_LineWidth is similar in that we can't guarantee anti aliased lines, and unfortunately its a pipeline setting so we'd need a new PSO per width which isn't practical